test/e2e/storage_test.go

```diff
@@ -96,6 +86,37 @@ func testStorage(t *testing.T) {
 			}
 		}()
 	}
+	if inTreeUpgrade {
+		err = tc.labelPodsNode(winServerDeployment.Spec.Selector.MatchLabels, map[string]string{storageTestLabel: "true"})
+		require.NoError(t, err)
+	}
+}
+
+// labelPodNode labels the Node which has the pod with matchLabels scheduled to it. Throws an error if more than one pod
+// matches the labels.
+func (tc *testContext) labelPodsNode(matchLabels map[string]string, labelsToApply map[string]string) error {
+	if matchLabels == nil {
+		return fmt.Errorf("nill matchLabels")
```

Comment:
nit: nil matchLabels or no labels present

Reply:
The check is only to ensure that a nil pointer deference doesn't happen which will cause a panic

Reply:
nit: `nill` --> `nil`